### PR TITLE
Linear regression

### PR DESCRIPTION
**Pull request by thejjserg**
Hello, Team - I found a more elegant method for calculating linear regression using matplot, scipy, and stats.

I've kept my trial-and-error method for calculating linear regression as I'll write a paragraph explaining it, and I'll be comparing the two methods.

### CODE REVIEW

File: us-medical-insurance-costs.ipynb [hunk 1]
```diff
@@ -1365,9 +1367,71 @@
...
+    "    # Outputs\n",
+    "    plt.show() \n",
+    "    print(\"The Linear Regression Line is \")\n",
+    "    print(\"Charges = \" + str(round(slope, 2)) + \" * \" + str(x_word.title()) + \" + \" + str(round(intercept, 2)))\n",
+    "    print(\"R is \" + str(round(r, 3)))\n",
+    "    print(\"Standard Error is \" + str(round(std_err, 3)))\n",
+    "    return [slope, intercept, r, p, std_err]\n",
     "\n",
-    "# Team! We are limited by the constraints"
+    "simple_linear_regression_age = simple_linear_regression(x = age, x_word = \"age\")"
```
Comment (GALLP900) [inline us-medical-insurance-costs.ipynb]: Is it possible to extract the name directly from the list instead of specifying it?

**Comment by GALLP900** [comment]
@thejjserg I think the linear regression works amazingly. nice work!! I did a simple test by plotting each variable with itself. this should give a perfect linear fit with R=1, 45deg slope and 0 offset or intercept which works 100% ok. My only comment would be when plotting any x,y variables not only x vs changes. Changes are needed in various lines of the function(s). Would be nice to be able to only specify the x y variables and have the plots to extract the labels directly from the list for example.  
Otherwise, this looks amazing, if @sunilyramjitsingh you dont have more comments we could merge this branch

**Comment by thejjserg** [comment]
@GALLP900 great comments! I've added your suggestions and added some changes. I found a function that someone wrote that get the name of a variable and returns it as a string, so that's helpful!

**Review by GALLP900 — APPROVED** [review]
Looks good, thanks for the improvements

**Pull request merged**